membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -211,7 +212,7 @@ def test_get_z_surface(x_bin, y_bin, x_range, y_range, expected_surface):
 class TestMembraneCurvature(object):
     @pytest.fixture()
     def universe(self):
-        return mda.Universe(GRO_PO4_SMALL, XTC_PO4_SMALL)
```

Comment:
I deleted the trajectory for now since I would need to create a new trajectory test with `u.dimensions[2]=30`. Skipping the trajectory the tests pass.